zerver/lib/user_topics.py

```diff
@@ -147,26 +148,19 @@ def set_user_topic_visibility_policy_in_database(
     )
 
     if created:
-        return
+        return skip_send_event
 
     duplicate_request: bool = row.visibility_policy == visibility_policy
 
-    if duplicate_request and ignore_duplicate:
-        return
+    if duplicate_request:
+        skip_send_event = True
+        return skip_send_event
```

Comment:
For this case, we may want to log (https://chat.zulip.org/#narrow/stream/378-api-design/topic/duplicate.20request.20to.20update.20UserTopic.20row/near/1522640)